lib/model/content.dart

```diff
@@ -438,22 +438,24 @@ abstract class EmojiNode extends InlineContentNode {
 }
 
 class UnicodeEmojiNode extends EmojiNode {
-  const UnicodeEmojiNode({super.debugHtmlNode, required this.text});
+  const UnicodeEmojiNode({super.debugHtmlNode, required this.emojiName, required this.emojiUnicode});
 
-  final String text;
+  final String emojiName;
+  final String? emojiUnicode;
```

Comment:
Yeah, after reading the surrounding code a bit more, it seems like it would be unreachable.
In previous revision, when `tryParseEmojiCodeToUnicode` returned null, parser would set `emojiUnicode` to null too. And in the widgets layer the `emojiName` with the fallback container would be displayed.

But thinking about it a bit more; if `tryParseEmojiCodeToUnicode` fails then we should just emit "unimplemented" from the parser rather than just adding a silent workaround. And also it will be pretty rare that `tryParseEmojiCodeToUnicode` would fail as it is mostly unreachable because of the regexp check in parser and if it does fail it would be a bug in the server.